evil-states.el

```diff
@@ -99,6 +99,8 @@ commands opening a new line."
                  evil-open-below
                  evil-append
                  evil-append-line
+                 evil-change
+                 evil-change-line
```

Comment:
I'd also be inclined to add `evil-change-whole-line` and any others which you notice are the same in vim.